<launch>

    <node pkg="object_detector" type="object_detector_node" name="object_detector_node" output="screen">
        <rosparam command="load" file="$(find object_detector)/config/simulation.yaml" />
    </node>

</launch>